package.json

```diff
@@ -44,6 +47,7 @@
     "mkdirp": "^0.5.1",
     "pako": "1.0.6",
     "query-string": "^6.1.0",
+    "renamer": "^0.7.3",
```

Comment:
These dependencies are required for cross-environment environment variable setting and renaming files. It'd be nice to do without them.